ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -508,6 +508,51 @@ tests:
     cluster_profile: openstack-vh-mecha-central
     workflow: openshift-e2e-openstack-external-lb
   timeout: 6h0m0s
+- always_run: false
+  as: e2e-openstack-masterscale
+  optional: true
+  steps:
+    cluster_profile: openstack-vexxhost
+    env:
+      TEST_SUITE: experimental/reliability/minimal
+    test:
+    - as: openstack-test-cpms
+      commands: |
+        dir=$GOPATH/src/github.com/openshift/cluster-control-plane-machine-set-operator
+        git clone https://github.com/openshift/cluster-control-plane-machine-set-operator --branch main --single-branch $dir
```

Comment:
Nit: to minimise resource consumption, you could set [`--depth=1`](https://git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt) (which by the way implies `--single-branch`) in your clones

Reply:
nice, I'll do that. Thanks for the suggestion.